### PR TITLE
fix(TBD-14205): fix CVE on commons-configuration2 and reload4j

### PR DESCRIPTION
**Pull request by AlixMetivier**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Review by dicarcab — DISMISSED** [review]

File: main/plugins/org.talend.libraries.neo4j/pom.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
 								<artifactItem>
 									<groupId>org.neo4j.talend</groupId>
 									<artifactId>neo4j-talend-component</artifactId>
-									<version>1.3-20191012</version>
+									<version>1.4-20220912</version>
```
Comment (lbourgeois) [inline main/plugins/org.talend.libraries.neo4j/pom.xml:35]: Is this change expected ?

Comment (AlixMetivier) [reply main/plugins/org.talend.libraries.neo4j/pom.xml:35]: it was done by previous PR so I guess so

Comment (AlixMetivier) [reply main/plugins/org.talend.libraries.neo4j/pom.xml:35]: @pyzhou-talend this was from original commit, can you confirm ?

Comment (pyzhou-talend) [reply main/plugins/org.talend.libraries.neo4j/pom.xml:35]: Hi @AlixMetivier 
I don't know why you create your branch based on mine (or cherry-pick my commit into your branch). But my change is not yet validated by QA and has not merged https://github.com/Talend/tbd-studio-se/pull/2884.
So this commit should not be merged into any master/maintenance branch.

Comment (pyzhou-talend) [reply main/plugins/org.talend.libraries.neo4j/pom.xml:35]: Please revert my commit from your branch.

**Review by dicarcab — APPROVED** [review]

**Pull request merged**